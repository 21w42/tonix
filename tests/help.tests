help t
help --help
help help
help
man help
quit

