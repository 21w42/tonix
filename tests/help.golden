Welcome to Tonix.
Type "help" to get a list of commands.
"man <COMMAND>" or "help <COMMAND>" sometimes might be helpful.
Some options for certain commands work as well.
Feel free to navigate a pre-made file system using intuitive commands.
Path resolution does not work yet, one step at a time please.
Your feedback is highly appreciated!
Have fun :)
$ 
help: no help topics match 't' 
Try 'help help' or 'man -k t' or 'info t' 

$ Usage: 	help 


	-d		output short description for each topic
	-m		display usage in pseudo-manpage format
	--help		display this help and exit
	--version	output version information and exit

$ Usage: 	help 


	-d		output short description for each topic
	-m		display usage in pseudo-manpage format
	--help		display this help and exit
	--version	output version information and exit

$ Commands: res0 basename cat cd chgrp chmod chown cksum cmp cp dd df dirname du echo file help ln ls man mkdir mv paste pwd rm rmdir stat touch uname wc whoami mount ping account 

$ help		User Commands
NAME
	help - help
SYNOPSIS
	help 
DESCRIPTION
	
	-d		output short description for each topic
	-m		display usage in pseudo-manpage format
	--help		display this help and exit
	--version	output version information and exit

$ Bye.
